.github/workflows/backwards-compatibility.yml

```diff
@@ -0,0 +1,37 @@
+name: Backwards compatibility
+
+on:
+    pull_request:
+
+concurrency:
+    group: bc-${{github.event_name}}-${{ github.head_ref || github.run_id }} # will be canceled on subsequent pushes in pull requests but not branches
+    cancel-in-progress: true
+
+jobs:
+    roave-backwards-compatibility-check:
+        name: Roave Backwards Compatibility Check
+        runs-on: ubuntu-latest
+
+        if: github.event.pull_request.draft == false
+
+        steps:
+            -   uses: actions/checkout@v3
+                with:
+                    token: ${{ secrets.BOT_TOKEN }}
+                    fetch-depth: 0
+
+            -   name: Setup PHP
+                id: setup-php
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.2'
+                    extensions: gd, intl, imagick
+                    coverage: none # disable xdebug, pcov
+                    
+            -   name: Install roave/backward-compatibility-check.
+                run: |
+                    composer global require roave/backward-compatibility-check
+                    echo "$(composer global config bin-dir --absolute --quiet)" >> $GITHUB_PATH
```

Comment:
Was hat es mit der Zeile auf sich? Ist die notwendig?

Reply:
Damit wird der globle bin dir in den path aufgenommen damit das tool aufrufbar ist